bigbluebutton-client/src/org/bigbluebutton/modules/users/views/MediaItemRenderer.mxml

```diff
@@ -96,6 +96,10 @@
 					rolledOver = true;
 					updateButtons();
 				}
+				else if (UserManager.getInstance().getConference().amIThisUser(e.userID) && (e.userID == data.userID)) {
```

Comment:
Would it not be better to change the previous condition to:

if ((moderator || UserManager.getInstance().getConference().amIThisUser(e.userID)) && e.userID == data.userID) {

since the inner code is the same?